webservices/schemas.py

```diff
@@ -372,27 +372,26 @@ class AuditCommitteeSearchListSchema(ApiSchema):
     }
 )
 
-
 augment_models(
     make_candidate_schema,
     models.Candidate,
     models.CandidateDetail,
 
 )
-#built these schemas without make_candidate_schema, as it was filtering out the flags
+# built these schemas without make_candidate_schema, as it was filtering out the flags
```

Comment:
I know that this was pre-existing, but any idea what this comment means?

Reply:
@vrajmohan at the engineering sync we agreed this should get refactored. Because this is a high-priority issue, we agreed that we should move forward with this PR and create a new issue for refactoring this endpoint: https://github.com/fecgov/openFEC/issues/3023